nixos/roles/mysql.nix

```diff
@@ -10,6 +10,7 @@ let
   fclib = config.fclib;
   supportedPerconaVersions = ["8.0" "8.3" "8.4"];
   removeDot = builtins.replaceStrings ["."] [""];
+  lokiServer = fclib.findOneService "loki-collector";
```

Comment:
This is an implementation detail which shouldn't need to be visible for roles which provide Alloy/Loki integration, but that's something we can solve separately later.